components/ContributorPopup.vue

```diff
@@ -83,23 +90,69 @@
         >
           {{ contributor.bio }}
         </div>
+
+        <div
+          v-if="contentId === 'category' && selectedCategory"
+          class="contributor-modal-content-contributions"
+        >
+          <ul>
+            <li
+              v-for="(repository, key) of selectedCategory.repositories"
+              :key="key"
+            >
+              <a
+                :href="`https://github.com/${key}/commits?author=${contributor.login}`"
+                target="_blank"
+                class="contributions-item repository"
+              >
+                <p class="contribution-number">{{ repository }}</p>
+
+                <p class="contribution-name">
+                  {{ key.replace('PrestaShop/', '') }}
+                </p>
+              </a>
+            </li>
+          </ul>
+        </div>
       </div>
     </div>
   </div>
 </template>
 
 <script lang="typescript">
-  import {BAvatar, BIconGeoAltFill, BIconX} from 'bootstrap-vue'
+  import {BAvatar, BIconGeoAltFill, BIconX, BIconArrowLeftShort} from 'bootstrap-vue'
   import ContributorRoles from './ContributorRoles'
   import ContributorLinks from './ContributorLinks'
   import CopyButton from './CopyButton'
 
+  const categoriesDatas = {
+    core: {
+      text: 'Contributions to the <strong>core</strong>'
+    },
+    documentation: {
+      text: 'Contributions to the <strong>documentation</strong>'
+    },
+    modules: {
+      text: 'Contributions to the <strong>modules</strong>'
+    },
+    others: {
+      text: 'Contributions to the <strong>others</strong>'
+    },
+    specs: {
+      text: 'Contributions to the <strong>specs</strong>'
+    },
+    tools: {
+      text: 'Contributions to the <strong>tools</strong>'
+    }
+  }
+
     export default {
```

Comment:
Weird indent here, look like there are two more spaces :thinking:

Reply:
I don't know why the linter doesn't fix this, but it's fixed!